src/components/Viewer.tsx

```diff
@@ -69,10 +83,40 @@ export default function Viewer({ handleTimeChange }: ViewerProps): ReactNode {
         }
     }, [page, setViewportToContainerSize, heightResized, trajectoryName]);
 
+    useEffect(() => {
+        const el = container.current;
+        el?.addEventListener("mousedown", (event) => {
+            if (!userHasInteracted) {
+                setFirstMousePosition({
+                    x: event.clientX,
+                    y: event.clientY,
+                });
+            }
+        });
+        container.current?.addEventListener("mouseup", (event) => {
+            if (
+                firstMousePosition.x !== event.clientX &&
+                firstMousePosition.y !== event.clientY
+            ) {
+                setUserHasInteracted(true);
+            }
+        });
+
+        el?.addEventListener("wheel", () => setUserHasInteracted(true));
+    }, []);
+
     if (!simulariumController) {
         return null;
     }
 
+    const hintOverlay = (
+        <div className={styles.hintOverlay}>
+            Play, scroll or click + drag to interact
```

Comment:
Just to check, playback won't make the interaction warning go away, because you want users to rotate/zoom/interact with the 3D scene?

Reply:
yes, everybody presses play, but they often just watch it instead of zooming in